components/bl-dropzone-component/src/index.js

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import { Dropzone, FileItem, FullScreenPreview, VideoPreview } from './lib/dropzone-ui.min';
+import { ensureMeasure, useDropzone } from './helpers';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function DropzoneComponent({ component, eventHandlers }) {
+  const {
```

Comment:
too many props, please break them into a few logical lines/blocks and then add an empty line in front of `eventHandlers `

```
const { display, classList, style, ... } = component
const { footerVisibility, headerVisibility, ... } = component
const { borderWidth, borderStyle, borderColor, ... } = component
...

 const { onDrop, onChangeView, onClean } = eventHandlers;

```